pyfstat/injection_parameters.py

```diff
@@ -122,12 +122,12 @@ class InjectionParametersGenerator:
         with ``@custom_prior`` or the name of a ``scipy.stats`` random variable.
 
         * | If a user-defined function is used, such a function *must* take
-          | a ``generator`` kwarg as one of its arguments and use such a generator
-          | (``np.random.Generator`` type)
-          | to generate any required random number within the function.
-          | The ``generator`` kwarg is *required* regardless of whether this is a
-          | deterministic or random function.
-          | For example, a negative log-distributed random number could be constructed as
+           | a ``generator`` kwarg as one of its arguments and use such a generator
+           | (``np.random.Generator`` type)
+           | to generate any required random number within the function.
+           | The ``generator`` kwarg is *required* regardless of whether this is a
+           | deterministic or random function.
```

Comment:
old help text, but I found it confusing on re-read. Maybe clearer:
```suggestion
           | to generate any required random number within the function.
           | (A deterministic function could then simply ignore this kwarg.)
```